web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java

```diff
@@ -90,22 +90,19 @@ public class QueryMetricsBean {
     public void updateMetric(BaseQueryMetric metric) throws Exception {
         DatawavePrincipal dp = getPrincipal();
         
-        if (metric.getLastWrittenHash() != metric.hashCode()) {
-            metric.setLastWrittenHash(metric.hashCode());
-            try {
-                metric.setLastUpdated(new Date());
-                sendQueryMetric(dp, metric);
-                // PageMetrics now know their own page numbers
-                // this should keep large queries from blowing up the queue
-                // Leave the last page on the list so that interceptors can update it.
-                Iterator<PageMetric> itr = metric.getPageTimes().iterator();
-                while (metric.getPageTimes().size() > 1) {
-                    itr.next();
-                    itr.remove();
-                }
-            } catch (Exception e) {
-                log.error(e.getMessage(), e);
+        try {
```

Comment:
Why are we getting rid of the hash check?

Reply:
@billoley removed that method in the latest query metrics service.  I talked to him about it when I noticed, and this was the change he recommended.